apps/gateway/src/routes/api/v3/conversations/index.ts

```diff
@@ -1,17 +1,6 @@
 import { createRouter } from '$/openApi/createApp'
 import { chatRoute, chatHandler } from './chat'
-import {
-  createEvaluationResultHandler,
-  createEvaluationResultV2Route,
-  createEvaluationResultV3Route,
-} from './createEvaluationResult'
-import { evaluateHandler, evaluateV2Route, evaluateV3Route } from './evaluate'
 
-const router = createRouter()
-  .openapi(chatRoute, chatHandler)
-  .openapi(evaluateV2Route, evaluateHandler)
-  .openapi(evaluateV3Route, evaluateHandler)
-  .openapi(createEvaluationResultV2Route, createEvaluationResultHandler)
-  .openapi(createEvaluationResultV3Route, createEvaluationResultHandler)
```

Comment:
I guess it's fine no one will complain. But should this be a new API version in the future for this kind of change?

Reply:
we don't even support this type of evals so api versioning wouldn't help anywayu